source/common/http/access_log/request_info_impl.h

```diff
@@ -54,6 +61,7 @@ struct RequestInfoImpl : public RequestInfo {
   uint64_t response_flags_{};
   Upstream::HostDescriptionConstSharedPtr upstream_host_{};
   bool hc_request_{};
+  std::string downstream_address_;
```

Comment:
Can we avoid keeping a copy of downstream_address_ in both here as well as the connection manager. I would probably just have it here, delete from connection manager, then allow setting via a public `setDownstreamAddress` which is not actually part of the interface but is public to the connection manager.